kubescaler/scaler/aws/cluster.py

```diff
@@ -553,6 +704,38 @@ def set_subnets(self):
                 self.vpc_subnet_public = output["OutputValue"].split(",")
             if output["OutputKey"] == "SubnetsPrivate":
                 self.vpc_subnet_private = output["OutputValue"].split(",")
+    
+    def wait_for_stack_updates(self):
+        while True:
+            stack_update = self.cf.describe_stacks(
+                StackName=self.workers_name
+            )
+            current_status = stack_update["Stacks"][0]["StackStatus"]
+            if "PROGRESS" in current_status:
+                print(f"The stack-{self.workers_name} is {current_status}")
+                time.sleep(5)
+                continue
+            else:
+                print(f"The stack-{self.workers_name} is {current_status}")
+                break
+        return 1
```

Comment:
Let's get rid of this return for 1, it's not being used. And since you continue, you shouldn't need the else. If progress isn't in the current_status, it will go beyond that if and you can do the print and break.